target/linux/ath79/generic/base-files/etc/uci-defaults/10_discard_oem_config

```diff
@@ -0,0 +1,10 @@
+
```

Comment:
Please add a comment which explains that this is needed to detect the OEM UCI configuration as otherwise they would be retained (more thouroughly).